aws/crr/sync_map.go

```diff
@@ -0,0 +1,27 @@
+package crr
+
+import (
+	"sync"
+)
+
+type syncMap sync.Map
+
+func newSyncMap() syncMap {
+	return syncMap{}
+}
+
```

Comment:
Is this file/type needed anymore?  Since SDK's min version includes `sync.Map` it may not be.